modules/udf/README.md

```diff
@@ -16,11 +16,45 @@ module "dataset" {
   location                    = "US"
 }
 
+local {
```

Comment:
It might be helpful to have an example showing the ability to read from a file (like in the `./examples` dir) for anyone that is coming from BQ world that is new to TF that might not know you can read from files.